interactive_sdk.py

```diff
@@ -20,7 +20,7 @@
 DISCOVERY_INTENTS = os.environ.get('DISCOVERY_INTENTS', '').split(',')
 
 # None here prints all datafram rows
-pd.set_option('display.max_rows', int(os.environ.get('DISCOVERY_DISPLAY_ROWS')))
+pd.set_option('display.max_rows', int(os.environ.get('DISCOVERY_DISPLAY_ROWS',0)))
```

Comment:
This will undo the change if the environment variable is set (note the comment above). We need this to be `None` for no all dataframes to print by default.
```suggestion
display_rows = int(os.environ.get('DISCOVERY_DISPLAY_ROWS',0))
pd.set_option('display.max_rows', display_rows if display_rows > 0 else None)
```